react/features/recording/components/Recording/web/StartRecordingDialog.js

```diff
@@ -19,6 +19,31 @@ import StartRecordingDialogContent from '../StartRecordingDialogContent';
  * @augments Component
  */
 class StartRecordingDialog extends AbstractStartRecordingDialog {
+
+    _onDisableStartRecording: () => boolean;
+
+    /**
+     * Disables start recording button.
+     *
+     * @returns {boolean}
+     */
+    _onDisableStartRecording() {
+        const { _fileRecordingsServiceEnabled, _isDropboxEnabled } = this.props;
+        const { isTokenValid, isValidating } = this.state;
+
+        // disable ok button id recording service is shown only, when
+        // validating dropbox token, if that is not enabled we either always
+        // show the ok button or if just dropbox is enabled ok is available
+        // when there is token
+        if (_fileRecordingsServiceEnabled) {
+            return isValidating;
+        } else if (_isDropboxEnabled) {
+            return !isTokenValid;
+        }
+
+        return false;
```

Comment:
Shouldn't this be true? If no recording service was configured then why should the start button be enabled?

Reply:
What about this case? You are saying that in this case the start button should be disabled? 
![Webp net-resizeimage](https://user-images.githubusercontent.com/26413496/159935593-3125c420-0589-4f11-ac70-0cd6ab040a46.jpg)

Reply:
There needs to be an extra check in case there is no file recording service configured, right?

Reply:
So, if the recording service is not configured, the button should be disabled?

Reply:
Nope, that would be local recording, which we don't have yet.

Reply:
Oooook. Now it's all clear for me. Thanks!